modules/og_context/og_context.info.yml

```diff
@@ -0,0 +1,7 @@
+name: OG context
+type: module
+description: Get a group from a viewed page.
+core: 8.x
+package: OG
+dependencies:
+  - og_ui
```

Comment:
Do you think this would have a strict dependency on OG UI?

I'm asking because in our project we do not need OG UI, but we will need OG Context, so it would be a pity if we have to enable an unused module for it.

This is probably to allow some configuration in the UI. We can make this optional as well, only unlock the UI when OG UI is enabled, or make an og_context_ui module :D

For me it is fine to keep it like this for now and tackle it later.